hands-on-exercise/3-cosmjs-adv/4-cosmjs-gui.md

```diff
@@ -258,12 +258,12 @@ For instance, prepare access to `rpcUrl` for `MenuContainer.tsx` by adding it to
     -  const App = () => {
     +  const App = ({ rpcUrl }: AppProps) => {
             ...
-            return (
-                <Container style={styles}>
+                <Routes>
+    -              <Route path="menu" element={<MenuContainer location={""} />} />
```

Comment:
BTW, here, is it 4 spaces or 1 `TAB`? It needs to be spaces.
Same for the second line?

Reply:
```suggestion
                <Routes>
    -                <Route path="menu" element={<MenuContainer location={""} />} />
```

Reply:
Added missing spaces to the <Route.../> line. 
<Routes> have spaces, not TAB.